src/generator/templates/Cargo.toml.mako

```diff
@@ -32,7 +32,7 @@ clap = "2"
 http-body-util = "0.1"
 % endif
 hyper = "1"
-hyper-rustls = { version = "0.27", default-features = false }
+hyper-rustls = { version = "0.27", default-features = false, features = ["http2", "rustls-native-certs", "native-tokio"] }
```

Comment:
How come these are enabled by default now?

Reply:
They were already enabled by `yup-oauth2` before, but since we now compile it without the `hyper-rustls` feature we have to enable it ourselves.